public/src/pages/views/MemberView.tsx

```diff
@@ -82,11 +81,11 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
                 "Registered users",
                 this.state.acceptedUsers,
                 [],
-                ActionType.Menu,
+                ActionType.InRow,
                 (user: IUserRelation) => {
                     const links = [];
                     if (user.link.getStatus() === Enrollment.UserStatus.TEACHER) {
-                        links.push({ name: "This is a teacher", extra: "primary" });
+                        links.push({ name: "Teacher", extra: "light" });
                     } else {
                         links.push({ name: "Make Teacher", uri: "teacher", extra: "primary" });
```

Comment:
Suggest to replace `Make Teacher` with `Appoint as Teacher`.
Also, perhaps make a little space between the buttons, so they are not touching.

Reply:
Are you sure about that? `Appoint as Teacher` button will take even more space.

Reply:
I'm leaning towards changing it to `Promote` only in the button. Then we could have a tool tip on the button?? with `Promote to teacher`, if possible?